src/backend/expungeservice/expunger/analyzers/type_analyzer.py

```diff
@@ -0,0 +1,70 @@
+class TypeAnalyzer:
+
+    def __init__(self):
+        self.class_b_felonies = []
+
+    def evaluate(self, charges):
+        for charge in charges:
+            self.__evaluate(charge)
+
+    def __evaluate(self, charge):
+        if charge.acquitted():
+            charge.expungement_result.set_type_eligibility(True)
+            charge.expungement_result.set_reason('Eligible under 137.225(1)(b)')
+
+        elif TypeAnalyzer.__marijuana_ineligible(charge):
+            charge.expungement_result.set_type_eligibility(False)
```

Comment:
There are 3 or 4 methods that take a `charge` object as an argument. These would be better placed in the charge class.